src/components/form-field/radio-box/radio-box-option.tsx

```diff
@@ -1,19 +1,20 @@
 import { RadioGroup } from "@headlessui/react";
 import React from "react";
 import { classNames } from "../../../util/class-names";
+import { RecommendationTag } from "../../recommendation-tag/recommendation-tag";
 
 export interface RadioBoxOptionProps {
     children: React.ReactNode;
     value: string;
     disabled?: boolean;
-    highlightText?: string;
+    recommendationText?: string;
 }
 
 export const RadioBoxOption = ({
     children,
     value,
     disabled,
-    highlightText,
+    recommendationText,
```

Comment:
If we follow the compound component pattern here, we could remove this prop and add a Radiobox.Option.Tag component here and add it as the first child but it does not really matter where because the position is absolut.